src/host/ledger.h

```diff
@@ -196,7 +196,10 @@ namespace asynchost
       DISPATCHER_SET_MESSAGE_HANDLER(
```

Comment:
Keeping this file around for now to have a clean diff for `multiple_ledger.h`. I will remove this one and rename `multiple_ledger.h` to `ledger.h` before merging this PR.